content/summits/developer/2025/_index.md

```diff
@@ -0,0 +1,154 @@
+authors: ["Brigitta Sipőcz", "K. Jarrod Millman", "Stéfan van der Walt"]
+date: 2024-03-05
+---
+
+## During the event
+
+Coming soon.
+
+## Agenda
+
+<style type="text/css">
+table {
+  margin-left: 0 !important;
+}
+</style>
+
+{{< details "**Saturday**" >}}
+| Time | Description |
+|------|-------------|
+| | Arrive |
+{{< /details >}}
+
+{{< details "**Sunday**" >}}
+| Time | Description |
+|------|-------------|
+| 09:00 | Meet at eScience |
+{{< /details >}}
+
+{{< details "**Monday**" >}}
+| Time | Description |
+|------|-------------|
+| 09:00 | Meet at eScience |
+{{< /details >}}
+
+{{< details "**Tuesday**" >}}
+| Time | Description |
+|------|-------------|
+| 09:00 | Meet at eScience |
+{{< /details >}}
+
+{{< details "**Wednesday**" >}}
+| Time | Description |
+|------|-------------|
+| 09:00 | Meet at eScience |
+{{< /details >}}
+
+</div>
+
+The event is held at the [eScience Institute, UW Physics/Astronomy Tower, 6th Floor, 3910 15th Ave NE, Seattle](https://goo.gl/maps/EfkoHtvZad3fYMx77).
+
+_A friendly reminder that we are guests of the eScience Institute, and
+that some members of their research staff will continue working in the space
+during our event._
+
+<br/>
+<iframe
+  src="https://www.google.com/maps/d/embed?mid=1eWqjU_k7dkYF8Z58sNQ9zJaYbR8HXPM&ehbc=2E312F"
+  width="100%" height="480"
+>
+</iframe>
+
+## Logistics
+
+We have space for 40 participants for three days and four nights.
+This will be an invite-only event that requires upfront agreement to:
+(a) take part in one or more pre-summit planning meetings,
+(b) collaborate with fellow participants on a work plan,
+(c) attend the summit in-person, and
+(d) participate, to whatever degree possible, in several months of post-summit implementation.
+
+### Airport to hotel
+
+There is no need to get a rental car or taxi/ride-share, as the airport and the UW campus area is well-served by bus and rail.
+We recommend using Link Light Rail to get from the airport to the hotel. It takes approximately an hour and costs $3.25.
+The closest station to the hotel is "U-District" (0.3mi away), which is a new station opened in 2021 and is not to be confused with the station in Downtown called, "University St", neither with "University of Washington".
+
+### Food
+
+Coming soon
+
+### Participants
+
+Participants are recruited from the community of developers of packages
+such as NumPy, SciPy, matplotlib, xarray, pandas, scikit-image, scikit-learn,
+NetworkX, and IPython, as well as domain stacks including Astropy, Pangeo, and
+scikit-HEP.
+
+{{< details "**List of participants** (we add names to the list as they confirm attendance)" >}}
+
+- Brigitta Sipőcz ([@bsipocz](https://github.com/bsipocz))
+- Jarrod Millman ([@jarrodmillman](https://github.com/jarrodmillman))
+- Stéfan van der Walt ([@stefanv](https://github.com/stefanv))
+
+{{< /details >}}
+
+## About
+
+### Goals
```

Comment:
We need to add a stats focusing section here, if not in this first iteration but in a follow-up update